app/src/main/java/com/example/daznsportslist/Adapter_RV.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.daznsportslist
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+
+class Adapter_RV(val playerList: List<String>) : Adapter<Adapter_RV.rv_ViewHolder>() {
```

Comment:
Use a better name for adapter, should be relevant

Reply:
done